low-level/files/opt/seagate/sspl/setup/generate_resource_health_view/resource_health_view

```diff
@@ -1302,16 +1302,23 @@ if __name__ == "__main__":
         print('Missing arguments')
         parser.print_help()
         sys.exit(1)
+    storage_type = Conf.get(GLOBAL_CONF, f"{STORAGE}>{ENCLOSURE}>{TYPE}",'virtual')
+    server_type = Conf.get(GLOBAL_CONF, f"{CLUSTER}>{SRVNODE}>{NODE_TYPE}",'virtual')
     print(f"Storage Type : '{storage_type}'")
     print(f"Server Type '{server_type}'")
 
+    if storage_type in ['5u84', 'rbod']:
```

Comment:
keep pods also in here. also level (lower or upper) the case for any input before comparing

Reply:
Done